Sources/StevensonCore/GHRepo.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Vapor
+
+typealias Version = String
+
+public struct GHRepo {
```

Comment:
Can we change `GH` to `GitHub`, or just `Repo`?
Or maybe it will make sense to introduce the name space `GitHub` (or just use `GithubService` for that?) and define `Repo` and `Release` inside it?
I can imagine that in GitLab or Bitbucket these types can be different. We could maybe even try to have a generic service for working with repositories that would work with any hosting. But that's of course out of scope of this PR.